examples/src/pages/Simple.tsx

```diff
@@ -30,7 +30,7 @@ export function Simple() {
         modelUrl: 'primitives',
       });
       const client = new CogniteClient({ appId: 'reveal.example.simple' });
-      client.loginWithOAuth({ project });
+      await client.loginWithOAuth({ type: 'CDF_OAUTH', options: { project }});
```

Comment:
Why doesn't this need `await client.authenticate();` ?

Reply:
It does :| I'll fix.